aspnetcore/fundamentals/minimal-apis/parameter-binding/samples8/Iform/Program.cs

```diff
@@ -43,17 +44,9 @@ async Task UploadFileWithName(IFormFile file, string fileSaveName)
 app.MapPost("/upload", async Task<Results<Ok<string>,
    BadRequest<string>>> (IFormFile file, HttpContext context, IAntiforgery antiforgery) =>
 {
-    try
-    {
-        await antiforgery.ValidateRequestAsync(context);
         var fileSaveName = Guid.NewGuid().ToString("N") + Path.GetExtension(file.FileName);
         await UploadFileWithName(file, fileSaveName);
         return TypedResults.Ok("File uploaded successfully!");
```

Comment:
This was wrapped in a try/catch block and when the try/catch code was removed, the indentation wasn't adjusted.